### PR TITLE
use sapi client >= 12.6

### PR DESCRIPTION
**Pull request by ErikZigo**
Part of https://keboola.atlassian.net/browse/PS-2638

### CODE REVIEW

File: tests/Functional/DownloadTablesOutputTest.php [hunk 1]
```diff
@@ -96,8 +96,8 @@ public function testDownloadTablesResult()
         $metrics = iterator_to_array($tablesResult->getMetrics()->getTableMetrics());
         self::assertEquals('in.c-input-mapping-test.test', $metrics[0]->getTableId());
         self::assertEquals('in.c-input-mapping-test.test2', $metrics[1]->getTableId());
-        self::assertGreaterThan(0, $metrics[0]->getUncompressedBytes());
-        self::assertSame(0, $metrics[0]->getCompressedBytes());
+        self::assertSame(0, $metrics[0]->getUncompressedBytes());
+        self::assertGreaterThan(0, $metrics[0]->getCompressedBytes());
```
Comment (ErikZigo) [inline tests/Functional/DownloadTablesOutputTest.php:100]: Since SAPI php client 12.6 table exports to files are with gzip compression.

Comment (odinuv) [reply tests/Functional/DownloadTablesOutputTest.php:100]: Supr, tak aspoň to vypadá, že to funguje :)

Comment (odinuv) [reply tests/Functional/DownloadTablesOutputTest.php:100]: no tak zpět - vyfailovalo to https://dev.azure.com/keboola-dev/input-mapping/_build/results?buildId=14678&view=logs&j=43d17d06-b74e-528d-cb87-c616d2d071eb&t=69b2550a-8e4f-5fcf-1aa5-7c03771d1b9a

Comment (odinuv) [reply tests/Functional/DownloadTablesOutputTest.php:100]: Starej build 🤦

**Pull request merged**